module/VuFind/src/VuFind/Hierarchy/Driver/Factory.php

```diff
@@ -68,35 +76,11 @@ public static function get(ServiceManager $sm, $config,
         $driverConfig = $configReader->get($config);
 
         // Build object:
-        return new $class(
+        return new \VuFind\Hierarchy\Driver\ConfigurationBased(
```

Comment:
You can just say ConfigurationBased here, because you're already in the VuFind\Hierarchy\Driver namespace.